src/Concordium/Client/Runner.hs

```diff
@@ -1379,7 +1404,9 @@ tailTransaction hash = do
     logFatal [ "transaction failed before it got committed"
              , "most likely because it was invalid" ]
 
-  runPrinter $ printTransactionStatus committedStatus False
+  schema <- getEventSchema False Nothing committedStatus Nothing
```

Comment:
Didn't the user set the `verbose` flag somewhere before they got to here? WHy are we setting it to `False`?

Reply:
It was set to false and thought this was the intended behaviour. I'll change it to match the value of the command line parameter?

Reply:
IIRC there was a reason for always using False, but I don't recall what it was. Let's use what the user specified, but please check that the output seems reasonable.